model.py

```diff
@@ -5303,8 +5298,8 @@ def sort_key(edition):
         return sorted(self.identifier.primarily_identifies, key=sort_key)
 
 
+    # TODO:  policy is not used in this method.  Remove argument?
```

Comment:
I take it back.  If I remove the policy argument, all tests break.  Keeping for now.  To do this in the future, I'd cut a separate branch.